pkg/ratelimit/runner.go

```diff
@@ -188,22 +193,25 @@ func (cr *ConcurrentRunner) RunTask(ctx context.Context, name string, f func(con
 
 	pendingTaskNum := len(cr.pendingTasks)
 	if pendingTaskNum > 0 {
+		if t, ok := cr.pendingRegionTasks[task.id]; ok {
+			t.f = f
```

Comment:
Can you add a comment to explain the function will be overwrite and it doesn't matter

rest lgtm